jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -305,6 +304,11 @@ def _aperture_selected_changed(self, event={}):
         else:
             self._background_selected_changed()
 
+    @property
+    def _cubeviz_slice_ind(self):
+        fv = self.app.get_viewer(self.app._jdaviz_helper._default_flux_viewer_reference_name)

```

Comment:
I don't know how this would be affected in the future when you allow multiple flux cubes and so on, but I guess we can worry about it in the future.

Reply:
Agreed - for now I was just trying to get the previous behavior in aperture photometry working with the new design.  This PR does not allow for multiple cubes, it just helps remove some assumptions in case we want to open that up later.